keyboards/viktus/sp111_v2/info.json

```diff
@@ -0,0 +1,179 @@
+{
+    "manufacturer": "BlindAssassin111",
+    "keyboard_name": "viktus/sp111_v2",
+    "maintainer": "blindassassin111",
+    "processor": "atmega32u4",
+    "bootloader": "atmel-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgb_matrix": true,
+        "rgblight": false
+    },
+    "keyboard_folder": "viktus/sp111_v2",
+    "keymaps": {},
+    "matrix_pins": {
+        "cols": ["C6", "F6", "F5", "F4", "F1", "F7", "B7", "D2", "D3", "D5", "D4"],
+        "rows": ["C7", "B6", "B5", "B4", "D7", "D6"]
+    },
+    "matrix_size": {
+        "cols": 11,
+        "rows": 12
+    },
+    "rgb_matrix": {
+        "driver": "WS2812",
+        "max_brightness": 150,
+        "layout": [
+        ]
+    },
+    "split": {
+        "enabled": true,
+        "main": "left",
```

Comment:
1. `main` currently not working, fix under way. work around is to define handedness in `config.h`
2. MASTER_LEFT is default handedness setting for split keyboards
```suggestion
```